examples/pysdl2.py

```diff
@@ -269,6 +297,18 @@ def main():
             sdl2.SDL_Rect(0, headerHeight, browserWidth, browserHeight)
         )
         sdl2.SDL_RenderPresent(renderer)
+
+        # FPS debug code
+        if logLevel == logging.DEBUG:
+            frames += 1
```

Comment:
My editor PyCharm displays two issues with "frames" and "lastFrameTick" variables ("local variable might be referenced before assignment"). Please move declaration of variables before the "if" block codes to get rid of these notices.

PyCharm editor has a free version and you can download cefpython configuratoin file for PyCharm in Issue #232.